src/features/shared-link-settings-modal/SharedLinkSettingsModal.js

```diff
@@ -303,15 +303,15 @@ class SharedLinkSettingsModal extends Component {
 
         return (
             message && (
-                <p className="link-settings-modal-notice">
+                <div className="link-settings-modal-notice">
```

Comment:
It looks like this change has removed some bottom margin, if you compare your screenshot to: http://opensource.box.com/box-ui-elements/storybook/?path=/story/features-sharedlinksettingsmodal--basic
Is that the intended change?

Reply:
Good catch! I updated the CSS rule and the image snapshot.